src/PluginBase/RadiumPluginInterface.hpp

```diff
@@ -146,7 +147,7 @@ class RadiumPluginInterface
      * MUST not be destroyed by the plugin
      */
     virtual void
-    addRenderers( std::vector<std::shared_ptr<Ra::Engine::Rendering::Renderer>>* /*rds*/ ) {}
+    addRenderers( std::vector<std::shared_ptr<Ra::Engine::Rendering::Renderer>>* rds ) {}
```

Comment:
If a parameter is not used in a function, the name should be omitted (https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-unused). 
If the name is given but the parameter is not used, several compilers generates a warning. 
So, either name the parameter in a comment to remind it or do not name the parameter. 
I prefer here the former as this declaration introduce a virtual function that should be overridden by derived classes.

Reply:
So, we should use the attribute `[[maybe_unused]]`. 
i.e.
```c++
virtual void
    addRenderers( [[maybe_unused]] std::vector<std::shared_ptr<Ra::Engine::Rendering::Renderer>>* rds ) {}
```
This will remove the compilation warning while allowing to document the parameter in the current doxygen version.
I prefer to not remove any warning but write the code in a proper way if there is a way to do so.